src/components/index.ts

```diff
@@ -3,3 +3,4 @@ export {default as FlightData} from './FlightData/FlightData';
 export {default as FlightInfo} from './FlightInfo/FlightInfo';
 export {default as Header} from './Header/Header';
 export {default as Icon} from './Icon/Icon';
+export {default as InputField} from './InputField/InputField';
```

Comment:
To be honest this entire file is just pointless, delete it. Were already exporting it directly from files with component